### ISSUE TITLE
CNN model parameters statistics or caculation

### ISSUE THREAD

**Issue by jingweimo** [fchollet/keras#7022]
I use Keras 1.1.1 (not yet switch to 2.0) for CNN modeling. The model has two convolution-pooling layers followed by two dense layers, and dropout is added following the second convolution-pooling layer and the first dense layer. The codes are as follows:

```
model = Sequential()
#convolution-pooling layers
model.add(Convolution2D(32, 5, 5, input_shape=input_shape))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Convolution2D(64, 5, 5))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))
#dense layers
model.add(Flatten()) 
model.add(Dense(100))
model.add(Activation('relu'))
model.add(Dropout(0.5))
model.add((Dense(2)))
model.add(Activation('softmax'))
#optimizer
sgd = SGD(lr=1e-3, decay=1e-6, momentum=0.9, nesterov=True) 
model.compile(loss='categorical_crossentropy',
              optimizer = sgd,
              metrics=['accuracy'])
print model.summary()

```
The model summary gives the table as follows:

Layer (type)                     Output Shape          Param #     Connected to   
convolution2d_11 (Convolution2D)       (None, 32, 76L, 76L)       832         convolution2d_input_6[0][0]      
activation_21 (Activation)                       (None, 32, 76L, 76L)       0           convolution2d_11[0][0]   
maxpooling2d_11 (MaxPooling2D)        (None, 32, 38L, 38L)       0           activation_21[0][0]            
convolution2d_12 (Convolution2D)        (None, 64, 34L, 34L)       51264       maxpooling2d_11[0][0]         
activation_22 (Activation)                        (None, 64, 34L, 34L)       0           convolution2d_12[0][0]       
maxpooling2d_12 (MaxPooling2D)         (None, 64, 17L, 17L)       0           activation_22[0][0]   
dropout_11 (Dropout)                             (None, 64, 17L, 17L)        0           maxpooling2d_12[0][0]    
flatten_6 (Flatten)                                     (None, 18496)                0           dropout_11[0][0] 
dense_11 (Dense)                                     (None, 100)                   1849700     flatten_6[0][0]    
activation_23 (Activation)                         (None, 100)                   0           dense_11[0][0]    
dropout_12 (Dropout)                              (None, 100)                   0           activation_23[0][0]    
dense_12 (Dense)                                    (None, 2)                        202         dropout_12[0][0]  
activation_24 (Activation)                        (None, 2)                         0           dense_12[0][0]         
Total params: 1901998

(I cannot insert a well-formatted table. I also post the issue at: https://stackoverflow.com/questions/44608552/keras-cnn-model-parameters-caculation)

About the table, I have  two confusions: 1) I am not clear of how the number of parameters of the second convolution layer (i.e., 51264) is computed. I thought the number would be (5*5 + 1)*64 = 1664, since the convolution kernel is 5 * 5 in size and 64 feature maps are to be extracted; 2) Besides, since I have already implemented dropout, why does the parameter table not reflect this point. It seems the parameter number without dropout is given.  

Anyone can help me with the problems?

Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by t-ae** [comment]
See this link
http://cs231n.github.io/convolutional-networks/#conv
> Every filter is small spatially (along width and height), but extends through the full depth of the input volume.

So the number of parameters is `5 * 5 * 32 * 64 + 64 = 51264`.
And `Dropout` has no parameters to learn.
